packages/encryption/src/encryptionDevice.ts

```diff
@@ -135,6 +157,34 @@ export class EncryptionDevice {
         )
     }
 
+    private async initializeFromExportedDevice(
```

Comment:
rewrite of [matrix implementation](https://github.com/matrix-org/matrix-js-sdk/blob/0fe53876ec555482bbeb086fd4dec8de7b2e21eb/src/crypto/OlmDevice.ts#L224), I wonder if the outbound/inbound sessions import here are the same of #257

Reply:
if thats true, this PR alone should close #603